package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "autoprefixer": "^9.8.5",
-    "docsearch.js": "^2.6.3",
+    "@docsearch/js": "^3.0.0-alpha.50",
```

Comment:
alpha? No stable release yet?

Reply:
Coming soon? https://github.com/algolia/docsearch/issues/1303
"We are actually looking at open issues to see if we encounter any blockers for the stable v3 release. The next release ideally be the stable one!"

Can we get a PR env for this? I am in contact with Algolia support, they will help us set up the search, and public  access to a working environment would be very helpful